src/abstract-command.js

```diff
@@ -198,10 +194,7 @@ class AbstractCommand {
    * @returns {String[]}
    */
   listAllEnvConfig(dir = false) {
-    return this._configLoader.listConfig({
-      dir: dir,
-      env: 'every'
-    });
+    return this._configLoader.listConfig({ dir: dir, env: Dictionary.ENVIRONMENT.SPECIFIC });
```

Comment:
value was `every`, but new constant is `Dictionary.ENVIRONMENT.SPECIFIC` ... is this correct?